src/skeleton/mod.rs

```diff
@@ -300,3 +304,34 @@ fn serialize_manifests(manifests: Vec<ParsedManifest>) -> Result<Vec<Manifest>,
     }
     Ok(serialised_manifests)
 }
+
+/// Remove all missing members from the root level `Cargo.toml` if it
+/// represents a workspace.
+///
+/// We judge whether a member is missing or not based on whether we were
+/// able to find its `Cargo.toml` file.
+fn remove_missing_members<P: AsRef<Path>>(
+    manifests: &[ParsedManifest],
+    base_path: P,
+) -> Result<(), anyhow::Error> {
+    let top_level_path = base_path.as_ref().join("Cargo.toml");
+    let contents = fs::read_to_string(&top_level_path)?;
+    let mut top_level: toml::Value = toml::from_str(&contents)?;
+    if let Some(members) = top_level
+        .get_mut("workspace")
+        .and_then(|workspace| workspace.get_mut("members"))
+    {
+        let members = members.as_array_mut().unwrap();
+        let found_members = manifests
+            .iter()
+            .filter_map(|manifest| manifest.relative_path.parent().map(|path| path.to_owned()))
```

Comment:
I'm a little worried if we run this from not the root directory. If this is a concern, I think that we can join both the members and the found members with the base path to be resilient to this? Maybe?